### DISCUSSION TITLE
Using schema instead of file paths in SQLs

### DISCUSSION THREAD

**Discussion by siripurapugopi** [ebyhr/trino-storage#168] [Q&A]
Is it possible to provide schema of files(csv, tsv, txt...etc) and use these for running SQLs. 

Instead of running `select * from storage.csv."/tmp/sales.csv"`, I want to run something like `select * from storage.my_db.sales`;

I tried creating a json file (etc/storage/sales.json, same attached as json.csv here) similar to what is mentioned under resources ([https://github.com/snowlift/trino-storage/blob/master/src/test/resources/example-data/example-metadata.json]). However I don't see any schema, table showing in Superset.

![image](https://user-images.githubusercontent.com/5980961/233773693-935a8360-6ddc-4799-8e6c-74ed884d34f1.png)
[sales.csv](https://github.com/snowlift/trino-storage/files/11300800/sales.csv)

**Comment by nineinchnick** [comment]
This is out of the scope of this plugin. It might be better to discuss this idea in the Trino Slack first.
